### PR TITLE
please review my pull-request. abdullah.is-a.dev

### PR DESCRIPTION
**Pull request by Adistar964**

### CODE REVIEW

File: domains/abdullah.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+{
+  "owner": {
+    "username": "adistar964",
+    "email": "adistar964@gmail.com"
+  },
+  "description":"this is for my personal website made.",
+  "record": {
+    "CNAME": "ali-developer.netlify.app"
```
Comment (TheBotlyNoob) [inline domains/abdullah.json:9]: We currently do not support Netlify or Vercel at the moment. The current workaround is to use [Railway](https://railway.app?referralCode=jj)

**Review by TheBotlyNoob — DISMISSED** [review]

File: domains/abdullah.json (hunk 1 above)
Comment (Adistar964) [reply domains/abdullah.json]: is gitub pages okay?

I changed it to github-pages link.

Please accept it.

**Comment by MaskDuck** [comment]
you don't need to get attentions by closing and re-opening.

**Review by MaskDuck — APPROVED** [review]
h

**Pull request merged**